src/example-types/facet.js

```diff
@@ -76,6 +105,9 @@ module.exports = {
               ]
             : []),
           _.get('label.fields', node) && projectStageFromLabelFields(node),
+          mapKeywordFilters(node),
+          { $sort: { count: -1 } },
+          node.size !== 0 && { $limit: node.size || 10 },
```

Comment:
If there is no options filter, it would be better to limit prior to the lookup. Merging as is because this fixes the bug, but let’s conditionally do the limit earlier when we can to improve performance